app/src/pages/inside/common/defectTypeItemML/defectTypeItemML.jsx

```diff
@@ -21,23 +21,29 @@ import styles from './defectTypeItemML.scss';
 
 const cx = classNames.bind(styles);
 
-export const DefectTypeItemML = ({ defectType, onClick, isSelected, className, isShownLess }) => {
+export const DefectTypeItemML = ({
+  defectType,
+  onClick,
+  isSelected,
+  className,
+  isShownLess: isShownMore,
```

Comment:
This is more confusing.
Why not call this prop just `isShowMore` and use it as expected?